src/encoded/static/components/app.js

```diff
@@ -58,7 +58,9 @@ const portal = {
                 { id: 'sep-mm-2' },
                 { id: 'publications', title: 'Publications', url: '/report/?type=Publication' },
             ],
-        },
+        }
+    ],
```

Comment:
You’ll get a couple ESLint errors here, one for not having a trailing comma and some for having tabs instead of spaces for indenting. I’m strict about those on encoded just to avoid having to write a ticket just to fix an ESLint error, which no one wants to do. But I don’t know if Lattice really cares about holding to ESLint rules or not.

Reply:
oh Emacs! I will double check my configs and update